cleverhans/attacks_tf.py

```diff
@@ -606,7 +631,7 @@ def attack(self, imgs, targets):
                                        targets[i:i + self.batch_size]))
         return np.array(r)
 
-    def attack_batch(self, imgs, labs):
+    def attack_batch(self, imgs, labs, mask=None):
```

Comment:
What will mask be used for? Also I don't see a reference to a mask described in the paper, I assume it's used in the case the attacker only wants to perturb a subset of the image.

Reply:
How come attack_batch uses the mask argument but attack doesn't?

Reply:
It's only needed for the L0 attack algorithm, so I only put it there for calling into it directly. I don't want to directly expose it to the outside world.